blas/unit_test/Test_Blas2_gemv.hpp

```diff
@@ -137,8 +145,13 @@ void impl_test_gemv(const char* mode, int M, int N) {
   numErrors = 0;
   for (int i = 0; i < ldy; i++) {
     if (KAT_Y::isNan(h_y(i)) ||
-        KAT_Y::abs(expected(i) - h_y(i)) > KAT_Y::abs(eps * expected(i)))
+        KAT_Y::abs(expected(i) - h_y(i)) >
+            KAT_Y::abs(alpha * max_valA * max_valX * ldx * eps * 2)) {
       numErrors++;
+      std::cout << "expected(" << i << ")=" << expected(i) << ", h_y(" << i
```

Comment:
One more print here. Do you want that merged?

Reply:
Yes, when there are errors I like actually knowing where they occurred.